etc/ci/buildbot_steps.yml

```diff
@@ -232,11 +232,31 @@ android-nightly:
 magicleap:
  env:
   MAGICLEAP_SDK: /Users/servo/magicleap/v0.17.0
+  MLCERT: /Users/servo/magicleap/certs/TempSharedCert.cert
  commands:
+  - unset OPENSSL_INCLUDE_DIR
+  - unset OPENSSL_LIB_DIR
   - ./mach clean-nightlies --keep 3 --force
   - ./mach clean-cargo-cache --keep 3 --force
   - ./etc/ci/clean_build_artifacts.sh
   - ./mach build --magicleap --dev
+  - /Users/servo/magicleap/v0.17.0/mabu -o ./target/release/magicleap -t lumin_debug ./support/magicleap/Servo2D/Servo2D.package
```

Comment:
I would rather have a `./mach package --magicleap` command here.

Reply:
I was going to hold off on a proper mach package command until we've worked on the app UI a bit more. Not a huge deal though.

Reply:
I want it to be as easy as possible to create a package to distribute to partners or for local testing.

Reply:
OK, it should be pretty easy to add to the package commands.

Reply:
Serves me right for saying "easy".